File: content/api-documentation/how-to/market-hours/examples/marketHours.go
package main

import (
	"fmt"
	"github.com/financefeasthq/financefeast-trade-api-go/financefeast"
)

func init() {
	financefeast.SetBaseUrl("https://paper-api.financefeast.markets")
}

func main() {
	// Check if the market is open now.
	clock, err := financefeast.GetClock()
	if err != nil {
		panic(err)
	}
	if clock.IsOpen {
		fmt.Println("The market is open.")
	} else {
		fmt.Println("The market is closed.")
	}

	// Check when the market was open on Dec. 1, 2018
	date := "2018-12-01"
	calendar, err := financefeast.GetCalendar(&date, &date)
	if err != nil {
		panic(err)
	}
	calendarDay := calendar[0]
	fmt.Printf("The market opened at %s and closed at %s on %s.\n",
		calendarDay.Open, calendarDay.Close, calendarDay.Date)
}

File: content/api-documentation/how-to/market-hours/examples/marketHours.js
const financefeast = require('@financefeasthq/financefeast-trade-api')
const financefeast = new financefeast()

// Check if the market is open now.
financefeast.getClock().then((clock) => {
    console.log('The market is ' + clock.is_open ? 'open.' : 'closed.')
})

// Check when the market was open on Dec. 1, 2018.
const date = '2018-12-01'
financefeast.getCalendar({
    start: date,
    end: date
}).then((calendars) => {
    console.log(`The market opened at ${calendars[0].open} and closed at ${calendars[0].close} on ${date}.`)
})